content/adventures/en.yaml

```diff
@@ -1428,7 +1428,7 @@ adventures:
                 story_text: |
                     Now we are going to change indentation a little bit. Every time that we need an indentation, we need `:` at the line before the indentation.
 
-                    In this level you can also use a new command: `{elif}`. `{elif}` is short for {else} {if} and you need it when you want to make 3 (or more!) options.
+                    In this level you can also use a new command: `{elif}`. `{elif}` is short for `{else} {if}` and you need it when you want to make 3 (or more!) options.
```

Comment:
Here too, are there untranslated languages with the same issue? In that case updating them there too would be lovely!

Reply:
yes of course, I will have a look.